code/extensions/Lumberjack.php

```diff
@@ -23,7 +23,7 @@ public function getExcludedSiteTreeClassNames()
         $classes = array();
         $siteTreeClasses = $this->owner->allowedChildren();
         foreach ($siteTreeClasses as $class) {
-            if (Config::inst()->get($class, 'show_in_sitetree') === false) {
```

Comment:
If its not `show_in_sitetree === false` then classes without `show_in_sitetree` set would start returning false here instead of true.